dataprofiler/labelers/data_processing.py

```diff
@@ -1507,7 +1508,7 @@ def process(  # type: ignore
         # with rework, can be tuned to be batches > size 1
         for ind in range(len(data)):
             batch_data: np.ndarray = data[ind : ind + 1]
-            batch_labels: Optional[List[str]] = (
+            batch_labels: Optional[npt.NDArray[np.str_]] = (
```

Comment:
why is this only `Optional[npt.NDArray[np.str_]]`, @Sanketh7 ? 

It appears that `self. convert_to_unstructured_format` takes in the `batch_labels` parameter as `list` or `npt.ndarray`? 

https://github.com/Sanketh7/DataProfiler/blob/mypy-flags/dataprofiler/labelers/data_processing.py#L1420

Reply:
updated